src/data/pnat/basic.lean

```diff
@@ -289,6 +289,9 @@ begin
   exact (this.trans h₀).symm,
 end
 
+theorem div_add_mod (m k : ℕ+) : (m : ℕ) = k * (div m k) + mod m k :=
+by rw add_comm; exact mod_add_div _ _
```

Comment:
```suggestion
(add_comm _ _).trans (mod_add_div _ _)
```

Reply:
I tried that locally and it didn't work for me. It complained about something which seemed like a problem with coercion.

Reply:
Seems to work in reverse order:
```suggestion
(mod_add_div _ _).trans (add_comm _ _)
```

Reply:
Right, that's why I was confused `pnat.div_add_mod` is stated in the opposite direction to the lemmas for `int` and `eudlidean domain`. Perhaps the direction of `mod_add_div` and `div_add_mod` should be reversed in this file?

Reply:
Thanks for looking into that. That was indeed strange. I think that change should be uncontroversial as pnat is rarely used.